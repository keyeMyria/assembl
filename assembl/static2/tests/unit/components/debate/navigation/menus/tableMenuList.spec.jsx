import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import TableMenuList from '../../../../../../js/app/components/debate/navigation/menus/tableMenuList';

describe('MenuList component', () => {
  it('should match the MenuList', () => {
    const items = [
      {
        id: 'foo',
        parentId: 'root'
      },
      {
        id: 'bar',
        parentId: 'root'
      }
    ];
    const props = {
      identifier: 'survey',
      className: 'debate-class',
      rootItem: 'root',
      items: items
    };
    const renderer = new ShallowRenderer();
    renderer.render(<TableMenuList {...props} />);
    const rendered = renderer.getRenderOutput();
    expect(rendered).toMatchSnapshot();
  });
});