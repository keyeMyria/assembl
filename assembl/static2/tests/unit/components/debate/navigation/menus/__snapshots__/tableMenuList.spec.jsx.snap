// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuList component should match the MenuList 1`] = `
<div
  className="menu-table-col debate-class"
>
  <div
    className="menu-table"
  >
    <Connect(DumbTableMenuItem)
      hasSubItems={false}
      identifier="survey"
      item={
        Object {
          "id": "foo",
          "parentId": "root",
        }
      }
      onClick={undefined}
      onMouseOver={[Function]}
      selected={false}
    />
    <Connect(DumbTableMenuItem)
      hasSubItems={false}
      identifier="survey"
      item={
        Object {
          "id": "bar",
          "parentId": "root",
        }
      }
      onClick={undefined}
      onMouseOver={[Function]}
      selected={false}
    />
  </div>
  
</div>
`;
